vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -115,14 +116,23 @@ As described in the [API reference](../API-reference/figure-callables.md) and il
 
 !!! example "KPI card variations"
 
+    === "my_css_file.css"
+
+        ```css
+        /* We define a fixed height and width to ensure uniform styling for all KPI cards. */
+        .flex-item .card-kpi {
+            width: 300px;
+            height: 150px;
+        }
+        ```
+
     === "app.py"
 
-        ```{.python pycafe-link}
+        ```py
         import pandas as pd
         import vizro.models as vm
         from vizro import Vizro
-        # For more information, refer to the API reference for kpi_card and kpi_card_reference
-        from vizro.figures import kpi_card, kpi_card_reference
+        from vizro.figures import kpi_card, kpi_card_reference  # (1)!
```

Comment:
Probably worth renumbering these to match the previous example once it's been corrected.